changelogs/fragments/parse_yum_output.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- related issue https://github.com/ansible/ansible/issues/70949
```

Comment:
no need a separate line, rahter put the issue URL at the end of the line hereunder between parenthesis (see other changelog files for example)